### ISSUE TITLE
Optional database

### ISSUE THREAD

**Issue by matheusho** [officert/mongotron#119]
Hi @officert.

Yesterday I did download of Mongotron and I think this is a awesome tool.

I've been testing and when I add another connection that do not be a localhost connection, the database name is required. Well, I understand that this is about security, but, in some cases, I have many databases in a unique connection.

I want to know if is possible the database name always be optional.

Thanks.

**Comment by officert** [comment]
Right now this is not possible. We are using the node mongodb driver under the hood which requires connecting a to a single database at a time.

**Comment by akatechis** [comment]
Hi there,

I just installed mongotron and noticed that the default connection "localhost" that exists when you first start the application has exactly this behavior (the "Database name" field is disabled, says "optional" and when I use it to connect, it shows all the databases on my local machine).

Oddly enough, I tried cloning that connection, but when I edit the host name, the database name field becomes enabled, and if I try to save the new connection, it highlights red.

Couldn't the application just re-use whatever logic the "localhost" connection is using to connect to all databases?
